Simulator/Layouts/Neuro/FixedLayout.cpp

```diff
@@ -129,11 +130,10 @@ void FixedLayout::loadParameters()
                           + inhibitoryNListFilePath);
    }
 
-   // Get the total number of vertices
-   int width, height;
-   ParameterManager::getInstance().getIntByXpath("//PoolSize/x/text()", width);
-   ParameterManager::getInstance().getIntByXpath("//PoolSize/y/text()", height);
-   numVertices_ = width * height;
+   // Get width, height and total number of vertices
+   ParameterManager::getInstance().getIntByXpath("//PoolSize/x/text()", width_);
+   ParameterManager::getInstance().getIntByXpath("//PoolSize/y/text()", height_);
```

Comment:
I think that we should move these parameters to the `<LayoutParams class="FixedLayout" name="LayoutParams">` section of the config files. If you like, create a new issue to do this, with a note of where in this file the change is needed (and, of course, that all the config files need changing).

Reply:
Created a new issue for this.